### DISCUSSION TITLE
Default note type

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [mortii/anki-morphs#100] [Ideas]
I had this idea for some time. But I thought it would be too complicated. But with the auto notetype fields it will work great.

Basically when someone installs ankimorph. It will create a note type and in the config it will have the default fields.

A pro of this is I want to add a feature in the future where we the user can select a book in .txt format or subtitles and automatically create cards with the default note type. I agree that subs2srs exists. This is for people who don't want or can't be bothered to have audio in their cards. I currentlyl use a regex to import a book into anki. But most people don't know about string parsing

**Comment by mortii** [comment]
Would this use anything from AnkiMorphsDB? If not then it should probably be a stand-alone add-on. Cool idea though.

**Comment by mortii** [comment]
@Vilhelm-Ian If you decide to make this then we can put a link to it in the tips & tricks section :)
